applications/serial_lte_modem/src/nativetls/slm_native_tls.c

```diff
@@ -27,7 +27,7 @@ struct tls_cred_buf {
 };
 static struct tls_cred_buf cred_buf[CONFIG_SLM_NATIVE_TLS_CREDENTIAL_BUFFER_COUNT] = {
 	[0 ... CONFIG_SLM_NATIVE_TLS_CREDENTIAL_BUFFER_COUNT - 1] = {
-		.sec_tag = -1
+		.sec_tag = 0xFFFFFFFF
```

Comment:
Can you clarify why we cannot add an include to get `NRF_SEC_TAG_TLS_INVALID` visible here?